githubteacher-hometown.md

```diff
@@ -1,5 +1,9 @@
-## Cupertino, CA
+## Fuquay Varina, NC
```

Comment:
I'm glad this is the name of a town.  I thought I was having a stroke when I read this.